jenkins/helper/test_launch_controller.py

```diff
@@ -776,21 +776,21 @@ def generate_crash_report(self):
         if IS_MAC:
             move_files = True
             system_corefiles = list(Path('/cores').glob(core_pattern))
-        files_unsorted = list(core_dir.glob(core_pattern)) + system_corefiles
+        files_unsorted = list(core_dir.glob(core_pattern))
+        if files_unsorted is None:
+            files_unsorted = []
+        files_unsorted += system_corefiles
```

Comment:
Can `system_corefiles` also be of type None, and what happens when appending it to `files_unsorted` here if it is? Do we need to check if `system_corefiles` is None here as well?
```
>>> files_unsorted = ["a", "b"]
>>> system_corefiles = None
>>> files_unsorted + system_corefiles
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can only concatenate list (not "NoneType") to list
```

Reply:
good catch, tnx.